lib/HTFeed/QueueRunner.pm

```diff
@@ -121,22 +121,7 @@ sub run_job_sequence {
         get_logger()->info(
             "next job: " . $job->namespace . "." . $job->id . " " . $job->stage_class
         );
-        # Get the verby part of the stage class name (e.g. "pack")
-        # that we need for job metrics.
-        my $jobtype    = lc((split("::", $job->stage_class))[-1]);
-        my $start_time = $self->{job_metrics}->time;
-
         $job->run_job($self->{clean});
-
-        # TODO: it's easy to count seconds and items for jobs out here
-        # but we have to go inside the job classes to count bytes r/w.
-        my $end_time       = $self->{job_metrics}->time;
```

Comment:
This was how and where a lot of metrics used to be set, "for free" by `QueueRunner` as it runs each stage, but that does not work if we want labels. For that we need to go inside each stage and look closer at the inputs/outputs. So this piece of code has to go, or we'll run the risk of counting things more than once.